node_modules/titanium-sdk/lib/emulators/genymotion.js

```diff
@@ -286,7 +286,10 @@ function findGenymotion(dir, config, callback) {
 	if (!player || !fs.existsSync(player)) {
 		player = path.join(dir, 'player' + exe);
 	}
-	if (player && !fs.existsSync(player)) {
+	if (!fs.existsSync(player) && process.platform != 'win32') {
```

Comment:
I'm fine with omitting the `player` truthiness checks. At least now it works on Linux.

I've slowly been migrating `==` to `===`. There's a lot of code. :)

Can you also please fix master?